functions.php

```diff
@@ -1,22 +1,12 @@
 <?php
 /**
- * Climate Hub Theme Functions
+ * Functions
  *
  * @package climatehub
  */
 
 require get_template_directory() . '/inc/cleanup.php';
 require get_template_directory() . '/inc/enqueue.php';
 require get_template_directory() . '/inc/theme-support.php';
-
-require get_template_directory() . '/inc/title-cf.php';
-require get_template_directory() . '/inc/paragraph-cf.php';
-require get_template_directory() . '/inc/highlightedSection-cf.php';
-require get_template_directory() . '/inc/content-right-side-img-cf.php';
```

Comment:
Are you working with the most up to date code? I already removed this code and the custom field php files.

Reply:
Master branch still has those lines which is why they’re showing up. I can double check tonight

Reply:
I know but my branch `refactor` shouldn't have any of those. My local copy doesn't have it and I have nothing else to commit.

Reply:
Nvm stupid questions, i figured it out